content/post/why-16k-page-size/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Why 16k page size matters
+description: 16k vs 4k page sizes & Asahi Linux.
+date: 2024-06-09 18:27:00 +0800
+---
+
+Since August of 2023, I've been a daily user of a 14-inch 2021 Apple MacBook Pro (M1 Pro, 16GB RAM) laptop.
+Thanks to [@tpwrules](https://github.com/tpwrules)'s [NixOS Asahi Linux Flake](https://github.com/tpwrules/nixos-apple-silicon/tree/main),
+I was able to set up the laptop to run NixOS. The majority of software have worked perfectly on it and I
+consider it to be the best Linux laptop for its performance. However, one major issue isn't a hardware issue
+but a software one. When you see memory sizes on a system, you commonly think about things like the timing,
+what kind of RAM chips they use, the number of RAM chips they use, and so on. Though, this isn't that.
```

Comment:
`number of RAM chips they use` have?

Reply:
Yes, the amount of RAM chips in a stick of memory or in each channel matters.